docs/csdl/TripService.xml

```diff
@@ -35,7 +35,15 @@
         <Property Name="Name" Type="Edm.String" />
         <Property Name="IcaoCode" Type="Edm.String" Nullable="false" />
         <Property Name="IataCode" Type="Edm.String" />
-        <Property Name="Location" Type="Microsoft.OData.Service.Sample.TrippinInMemory.Models.AirportLocation" />
+        <Property Name="Location" Type="Microsoft.OData.Service.Sample.TrippinInMemory.Models.AirportLocation">
```

Comment:
sorry, I meant [this one](https://github.com/microsoft/OpenAPI.NET.OData/blob/master/test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OData.xml)  (and update the generated results)

Reply:
@baywet  I have added some examples and updated the generated openapi documents. Thank you.